Src/Core/Equivalency/FieldSelectedMemberInfo.cs

```diff
@@ -30,5 +31,15 @@ public override Type MemberType
         {
             get { return fieldInfo.FieldType; }
         }
+
+        public override CSharpAccessModifier GetAccessModifier
```

Comment:
Rather than adding a property, wouldn't it not make more sense to replace them with methods, e.g. WhichSetterHasModifier(CSharpAccessModifier.Internal);